pkg/cmd/cli/cmd/status.go

```diff
@@ -59,11 +60,15 @@ type StatusOptions struct {
 // NewCmdStatus implements the OpenShift cli status command.
 func NewCmdStatus(name, fullName string, f *clientcmd.Factory, out io.Writer) *cobra.Command {
 	opts := &StatusOptions{}
+	cmdName := fullName
+	if strings.HasSuffix(fullName, name) {
```

Comment:
Yeah, `NewCmdStatus` should take an additional argument with what you want here (`oc|openshift cli|...`), ready to use. I *think* `fullName` gives you that (I mean `fullName` in `cli.go`, the one you have before appending `StatusRecommendedName`).